tests/system/test_diags.py

```diff
@@ -169,15 +169,22 @@ def check_html_image(self, html_path, png_path, full_png_path):
         path_to_actual_png = full_png_path
         path_to_expected_png = "../unit_test_images/{}".format(png_path)
 
-        mismatched_images = []  # type: ignore
-        compare_images(
-            self,
-            mismatched_images,
-            image_name,
-            path_to_actual_png,
-            path_to_expected_png,
-        )
-        self.assertEqual(mismatched_images, [])
+        try:
+            # Set `export CHECK_IMAGES=True` to do a pixel-by-pixel image comparison check.
+            check_images = bool(os.environ["CHECK_IMAGES"])
+        except KeyError:
+            check_images = False
```

Comment:
A suggestion based on https://stackoverflow.com/a/65407083/7728169

```suggestion
        if "CHECK_IMAGES" in os.environ:
            # Set `export CHECK_IMAGES=True` to do a pixel-by-pixel image comparison check.
            check_images = os.environ["CHECK_IMAGES"].lower() in ['true', 'yes', 't', 'y' '1']
        else:
            check_images = True
```

Reply:
It's generally better to do and `if ... in ..` than a `try ... except` in case where the former is a possibility.  It's both a bit easier to read and more specific.